astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -46,38 +52,65 @@ class Galactocentric(BaseCoordinateFrame):
         {\rm Dec} = -28:56:10.23~{\rm deg}
 
     The default distance to the Galactic Center is 8.3 kpc, e.g.,
-    Gillessen et al. 2009,
-    http://adsabs.harvard.edu/abs/2009ApJ...692.1075G.
+    Gillessen et al. (2009),
+    https://ui.adsabs.harvard.edu/#abs/2009ApJ...692.1075G/abstract
 
     The default height of the Sun above the Galactic midplane is taken to
-    be 27 pc, as measured by
-    http://adsabs.harvard.edu/abs/2001ApJ...553..184C.
+    be 27 pc, as measured by Chen et al. (2001),
+    https://ui.adsabs.harvard.edu/#abs/2001ApJ...553..184C/abstract
+
+    The default solar motion relative to the Galactic center is taken from a
+    combination of Schönrich et al. (2010) [for the peculiar velocity] and
+    Bovy (2015) [for the circular velocity at the solar radius],
+    https://ui.adsabs.harvard.edu/#abs/2010MNRAS.403.1829S/abstract
+    https://ui.adsabs.harvard.edu/#abs/2015ApJS..216...29B/abstract
 
     For a more detailed look at the math behind this transformation, see
     the document :ref:`coordinates-galactocentric`.
 
+    The frame attributes are listed under **Other Parameters**.
+
     Parameters
     ----------
-    representation : `BaseRepresentation` or None
+    representation : `~astropy.coordinates.representation.BaseRepresentation` or None
         A representation object or None to have no data (or use the other
         keywords)
+
+    x : `~astropy.units.Quantity`, optional
+        Cartesian, Galactocentric :math:`x` position component.
+    y : `~astropy.units.Quantity`, optional
+        Cartesian, Galactocentric :math:`y` position component.
+    z : `~astropy.units.Quantity`, optional
+        Cartesian, Galactocentric :math:`z` position component.
+
+    v_x : `~astropy.units.Quantity`, optional
+        Cartesian, Galactocentric :math:`v_x` velocity component.
+    v_y : `~astropy.units.Quantity`, optional
+        Cartesian, Galactocentric :math:`v_y` velocity component.
+    v_z : `~astropy.units.Quantity`, optional
+        Cartesian, Galactocentric :math:`v_z` velocity component.
+
+    copy : bool, optional
+        If `True` (default), make copies of the input coordinate arrays.
+        Can only be passed in as a keyword argument.
+
+    Other parameters
+    ----------------
+    galcen_coord : `ICRS`, optional, must be keyword
```

Comment:
In the other docstrings, the "must be keyword" is omitted (though in this case it is true). Perhaps easiest to omit here too, and have "other parameters" be keywords by default?